README.md

```diff
@@ -63,6 +63,21 @@ If so, it'll update the project and the dependencies.
 Then it'll build the frontend and copy it into the backend folder so it can be served.
 The server starts and you can access the app at [http://localhost:3000](http://localhost:3000).
 
+### Docker mode
+
+Build an pizza-sync image:
+```bash
+docker build -t pizza:latest .
+```
+
+Then run a container:
+```bash
+docker run -it -d --name pizza -p 8081:3000 pizza:latest
```

Comment:
I had a problem running this command.
I had to run it like that:
`docker run -p 3000:3000 -it pizza-sync`

Also, could you please call it `pizza-sync` instead of just `pizza`, thanks :)